### PR TITLE
feature/exception-handler

### PR DESCRIPTION
**Pull request by Nazar865**

### CODE REVIEW

File: Corzbank.Api/Startup.cs [hunk 1]
```diff
@@ -71,6 +65,8 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseHttpsRedirection();
 
+           app.ConfigureCustomExceptionMiddleware();
```
Comment (ghost) [inline Corzbank.Api/Startup.cs]: fix formatting (add a space)

File: Corzbank.Services/Interfaces/ILoggerManager.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+using System.Text;
+
+namespace Corzbank.Services.Interfaces
+{
+    public interface ILoggerManager
+    {
+        void LogInfo(string message);
+        void LogWarn(string message);
+        void LogDebug(string message);
+        void LogError(string message);
```
Comment (ghost) [inline Corzbank.Services/Interfaces/ILoggerManager.cs]: Add empty lines between methods for consistency

File: Corzbank.Services/LoggerManager.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            logger.Error(message);
+        }
+        public void LogInfo(string message)
+        {
+            logger.Info(message);
+        }
+        public void LogWarn(string message)
+        {
+            logger.Warn(message);
+        }
```
Comment (ghost) [inline Corzbank.Services/LoggerManager.cs]: Add empty lines between methods for consistency

File: Corzebank.Extensions/ExceptionMiddlewareExtension.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace Corzbank.Extensions
+{
+    public static class ExceptionMiddlewareExtension
+    {
+        public static void ConfigureCustomExceptionMiddleware(this IApplicationBuilder app)
```
Comment (ghost) [inline Corzebank.Extensions/ExceptionMiddlewareExtension.cs]: rename to ConfigureExceptionHandlingMiddleware

File: Corzbank.Api/Startup.cs (hunk 1 above)
Comment (Nazar865) [reply Corzbank.Api/Startup.cs]: resolved

File: Corzbank.Services/Interfaces/ILoggerManager.cs (hunk 2 above)
Comment (Nazar865) [reply Corzbank.Services/Interfaces/ILoggerManager.cs]: resolved

File: Corzbank.Services/LoggerManager.cs (hunk 3 above)
Comment (Nazar865) [reply Corzbank.Services/LoggerManager.cs]: resolved

File: Corzebank.Extensions/ExceptionMiddlewareExtension.cs (hunk 4 above)
Comment (Nazar865) [reply Corzebank.Extensions/ExceptionMiddlewareExtension.cs]: resolved

**Review by ghost — APPROVED** [review]

**Pull request merged**